packages/vite/src/node/plugins/css.ts

```diff
@@ -2020,6 +2023,24 @@ function loadPreprocessorPath(
   }
 }
 
+function loadSassPackage(root: string): {
+  name: 'sass' | 'sass-embedded'
+  path: string
+} {
+  // try sass-embedded before sass
+  try {
+    const path = loadPreprocessorPath('sass-embedded' as any, root)
+    return { name: 'sass-embedded', path }
+  } catch (e1) {
+    try {
+      const path = loadPreprocessorPath('sass' as any, root)
```

Comment:
Could we update the accepted type from `loadPreprocessorPath` instead of `as any` here?

Reply:
Fixed https://github.com/vitejs/vite/pull/17754/commits/81687e3c22a72f8ce71038ac932a90ea68d96b08